package.json

```diff
@@ -73,6 +73,7 @@
     "newrelic": "11.23.2",
     "pino": "^9.3.2",
     "pino-pretty": "^11.2.2",
+    "prom-client": "^15.1.3",
```

Comment:
this is not a new dependency, we get it from fastify-metrics anyway, but explicit listing allows to import from it